docs/features/interceptors.md

```diff
@@ -191,7 +191,12 @@ Interceptors are treated like a post-compilation step in this design. Diagnostic
 
 ### User opt-in
 
-Interceptors will require a feature flag during the experimental phase. The flag can be enabled with `/features=InterceptorsPreview` on the command line or `<Features>InterceptorsPreview</Features>` in msbuild.
+To use interceptors, the user project must specify the property `<InterceptorsPreviewNamespaces>`. This is a list of namespaces which are allowed to contain interceptors.
+```xml
+<InterceptorsPreviewNamespaces>$(InterceptorsPreviewNamespaces);Microsoft.AspNetCore.Http.Generated;MyLibrary.Generated</InterceptorsPreviewNamespaces>
+```
+
+It's expected that each entry in the `InterceptorsPreviewNamespaces` list roughly corresponds to one source generator. Well-behaved components are expected to not insert interceptors into namespaces they do not own.
```

Comment:
Should the `InterceptorsPreview` feature flag still be documented, since it's still used/implemented?

Reply:
No need to document the old flag but there is a need to keep it working till our consumers onboard to the new one. This avoids needing to dual insert into SDK, etc.

Reply:
Oh, I thought we'd keep the old flag for the scenario with interceptor without namespace. How will that scenario work?

Reply:
Tbf, I feel comfortable just breaking that scenario. If somebody opts in for the global namespace then we're back to global opt in. @jaredpar may want to weigh in

Reply:
> Oh, I thought we'd keep the old flag for the scenario with interceptor without namespace. 

This isn't a scenario that concerns me. Code generated into the global namespace is less reliable than code generated into a private namespace (more ambiguity issues to consider).